dnsaas/api/v2/serializers.py

```diff
@@ -81,6 +88,10 @@ class Meta:
         required=False,
         allow_null=True,
     )
+    service = PrimaryKeyRelatedField(
```

Comment:
what if I'll want to show service of the record on the main page (list of records)? I'll have to do separate query for every record, am i right?